packages/lib/src/components/Card/components/CardInput/components/DisclaimerMessage/DisclaimerMessage.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { shallow } from 'enzyme';
+import { h } from 'preact';
+import DisclaimerMessage from './DisclaimerMessage';
+
+describe('DisclaimerMessage', () => {
+    const disclaimerMessage = {
+        message: 'By continuing you accept the %{linkText} of MyStore',
```

Comment:
Maybe I am missing something, but did we also consider to provide the `message` label in our translations file?

Reply:
No. They want a minimal implementation on our side. 
It's up to the merchant to provide the translated text.